testsuite/features/support/commonlib.rb

```diff
@@ -176,6 +176,18 @@ def get_client_type(name)
   end
 end
 
+def suse_host?(name)
+  (name.include? 'sle') || (name.include? 'opensuse')
+end
+
+def rh_host?(name)
```

Comment:
We might need more soon. Like other RH like distros, Oracle Linux, Amazon Linux, and so on.

Reply:
That's the exact motivation for this PR.